core/src/banking_stage/forwarder.rs

```diff
@@ -189,14 +200,14 @@ impl Forwarder {
                     .forwarded_vote_count
                     .fetch_add(packet_vec_len, Ordering::Relaxed);
                 let pkts: Vec<_> = packet_vec.into_iter().zip(repeat(addr)).collect();
-                batch_send(socket, &pkts).map_err(|err| err.into())
+                batch_send(&self.socket, &pkts).map_err(|err| err.into())
             } else {
                 // All other transactions can be forwarded using QUIC, get_connection() will use
                 // system wide setting to pick the correct connection object.
                 banking_stage_stats
                     .forwarded_transaction_count
                     .fetch_add(packet_vec_len, Ordering::Relaxed);
-                let conn = connection_cache.get_connection(&addr);
+                let conn = self.connection_cache.get_connection(&addr);
```

Comment:
Had a conflict w/ #29774 here that needed resolution. All I added was the `self.`, since `connection_cache` belongs to the `Forwarder` now.

@lijunwangs or @taozhu-chicago could I get a re-approval so I can merge this once CI passes? Looks like it's formally required now 😄

Reply:
sorry didn't get to check GH notification till now. Looks Lijun had taken care of it.